tests/common/debug.py

```diff
@@ -76,7 +76,7 @@ def find_any(
     settings = Settings(
         settings,
         max_examples=10000,
-        max_shrinks=0000,
+        phases=[p for p in list(Phase) if p != Phase.shrink],
```

Comment:
util.no_shrink?

Reply:
Considered it, but I'd rather not add imports between test setup files for something this simple.